packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -970,3 +1061,12 @@ function extendBuildSpec(buildSpec: any, extend: any) {
     phase.commands.push(...extend.phases[phaseName].commands);
   }
 }
+
+function resourcePolicyStatementForRepository() {
+  return new iam.PolicyStatement()
+    .describe('CodeBuild')
+    .addServicePrincipal('codebuild.amazonaws.com')
+    .addAction('ecr:GetDownloadUrlForLayer')
+    .addAction('ecr:BatchGetImage')
+    .addAction('ecr:BatchCheckLayerAvailability');
```

Comment:
`addActions` here instead?

Reply:
Why?

Reply:
Because it's a lot less verbose...?

Reply:
I guess it's a matter of style and since you are effectively the owner of this module, you get to decide ;-)

Reply:
Do you seriously prefer this:

```ts
  return new iam.PolicyStatement()
    .describe('CodeBuild')
    .addServicePrincipal('codebuild.amazonaws.com')
    .addAction('ecr:GetDownloadUrlForLayer')
    .addAction('ecr:BatchGetImage')
    .addAction('ecr:BatchCheckLayerAvailability');
```

over this:

```ts
  return new iam.PolicyStatement()
    .describe('CodeBuild')
    .addServicePrincipal('codebuild.amazonaws.com')
    .addActions(
      'ecr:GetDownloadUrlForLayer',
      'ecr:BatchGetImage',
      'ecr:BatchCheckLayerAvailability',
    );
```

? I'm not trying to be sarcastic or anything, I'm genuinely curious, because for me the second one is clearly superior.